sirepo_bluesky/utils/__init__.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+
+
+def get_beam_stats(image, x_extent, y_extent):
+    n_y, n_x = image.shape
+
+    if image.sum() > 0:
```

Comment:
`image.sum()` is used here and in return. Let's calculate it once and use in all 6 places.